examples/packages/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
+
+VARIANTS = ["npm_json", "npm_lock"]
```

Comment:
this doesn't look much like an example for users to follow. I suspect there should be some e2e testing folder under `/internal` where this kind of exhaustive checking lives.

The better example would just be to convert `examples/program` to use bazel-managed-deps